keepalived/service/running.sls

```diff
@@ -12,6 +12,9 @@ include:
 keepalived-service-running-service-running:
   service.running:
     - name: {{ keepalived.service.name }}
-    - enable: True
     - watch:
       - sls: {{ sls_config_file }}
+
+keepalived-service-enabled:
```

Comment:
```suggestion
keepalived-service-running-service-enabled:
```
One small change I can suggest - change state id to match with formula naming scheme: `path-to-slsfile-state-name` yes it's look little bit weird, I didn't like it at first, but it helps to track which file state belongs.

Reply:
yes, it makes sense to keep them consistent.